packages/webapps/requirements.txt

```diff
@@ -2,6 +2,7 @@ galaxy-app
 Cheetah3
 fastapi>=0.68.2,!=0.69.0,!=0.70.0,!=0.70.1
 fastapi-utils
+importlib_resources
```

Comment:
This needs to be added to pyproject.toml as well.

Reply:
Thanks, I've pushed that into https://github.com/galaxyproject/galaxy/pull/13147/commits/0d28033aa2e5f2b6e756c7d03c64fa388f0e5cb5